spec.html

```diff
@@ -1939,31 +1939,31 @@ <h1>BigInt::sameValueZero ( _x_, _y_ )</h1>
         <emu-clause id="sec-binaryand" aoid="BinaryAnd">
           <h1>BinaryAnd ( _x_, _y_ )</h1>
           <emu-alg>
-            1. Assert: _x_ is 0 or 1.
-            1. Assert: _y_ is 0 or 1.
-            1. If _x_ is 1 and _y_ is 1, return 1.
-            1. Else, return 0.
+            1. Assert: _x_ is *0n* or *1n*.
```

Comment:
Hm, I'm not sure about this change. [BigIntBitwiseOp](https://tc39.es/ecma262/#sec-bigintbitwiseop), the caller of these operations, deals with unsuffixed numbers.

Reply:
Yeah, those should probably be changed as well (in this PR or not, I don't mind). (Presumably this PR didn't touch BigIntBitwiseOp because it doesn't have any occurrences of "is zero" or "is 0".)